README.md

```diff
@@ -161,33 +161,44 @@ Fields to return:
 - `available_inventory` (not currently checked-out to a customer)
   - This will be the same as `inventory` unless you've completed the optional endpoints.
 
+#### `POST /movies`
+Create a new movie in the video store inventory.
```

Comment:
Should we specify that this endpoint returns any particular information, such as the ID of the new movie?

Reply:
I didn't mostly because the other POST requests don't - but we could. We should probably do something similar for the rental endpoints but I'm not sure what those should return. Thoughts on that?